lexicons/app/bsky/feed/searchPosts.json

```diff
@@ -13,6 +13,53 @@
             "type": "string",
             "description": "Search query string; syntax, phrase, boolean, and faceting is unspecified, but Lucene query syntax is recommended."
           },
+          "sort": {
+            "type": "string",
+            "knownValues": ["top", "latest"],
+            "default": "latest",
+            "description": "Specifies the ranking order of results."
+          },
+          "since": {
+            "type": "string",
+            "description": "Filter results for posts after the indicated datetime (inclusive). Expected to use 'sortAt' timestamp, which may not match 'createdAt'. Can be a datetime, or just an ISO date (YYYY-MM-DD)."
+          },
+          "until": {
+            "type": "string",
+            "description": "Filter results for posts before the indicated datetime (not inclusive). Expected to use 'sortAt' timestamp, which may not match 'createdAt'. Can be a datetime, or just an ISO date (YYY-MM-DD)."
+          },
+          "mentions": {
+            "type": "string",
+            "format": "at-identifier",
+            "description": "Filter to posts which mention the given account. Handles are resolved to DID before query-time. Only matches rich-text facet mentions."
+          },
+          "author": {
+            "type": "string",
+            "format": "at-identifier",
+            "description": "Filter to posts by the given account. Handles are resolved to DID before query-time."
+          },
+          "lang": {
```

Comment:
Do we want to support multiple values for this one?

Reply:
for `from` I assume, with "OR" semantics?

I thought about it and leaned against, though not a strong opinion. I'm guessing that folks might want to try to hack in "list search" functionality using this. But these are query params, so only would work with a couple dozen accounts (?). I think the better functionality would be actual list search as a follow-up thing, where we fetch list membership server-side.